Add new reference module for expiring RHEL support (#2617)
theforeman/foreman-documentation: guides/common/modules/con_lifecycle-status-of-rhel-hosts.adoc

[id="lifecycle_status_of_rhel_hosts_{context}"]
= Lifecycle Status of RHEL Hosts

{Project} provides multiple mechanisms to display information about upcoming End of Support (EOS) events for your {RHEL} hosts:

* Notification banner
* A column on the Hosts index page
* *Search* field

> You can find the {EL} lifecycle status on the *Host status* card on the host details page.

Reviewer: ```suggestion
```

> * Alert on the Hosts index page for each host that runs {EL} with an upcoming EOS event in a year as well as when support has ended

Reviewer: ```suggestion
* Alert on the Hosts index page for each host that runs {RHEL} with an upcoming EOS event in a year as well as when support has ended
```
You might want to verify this with a SME, but are you sure that the lifecycle status other than Unknown can be displayed for example for CentOS hosts? CentOS is EL as well. If this for RHEL specifically, then let's be exact about it.
The same applies to other occurences.

Author: Double checking with SME.

Author: Confirmed it's just RHEL.

> For any hosts that are not running {EL}, {Project} displays *Unknown* in the *RHEL Lifecycle status* and *Last report* columns.

Author: I jumped the gun too fast! Fixing... :)

[id="lifecycle_status_of_rhel_hosts_{context}"]

> = Lifecycle Status of {RHEL} Hosts

Reviewer: ```suggestion
= Lifecycle Status of RHEL Hosts
```
I'd use just the abbreviation in the title.

> {Project} provides multiple mechanisms to display information about upcoming End of Support (EOS) events for your {RHEL} clients.

Reviewer: ```suggestion
{Project} provides multiple mechanisms to display information about upcoming End of Support (EOS) events for your {RHEL} hosts:

* Notification banner
* A column on the Hosts index page
* etc.
```
Would it be a good idea to list the mechanisms before you go into details?

> 

Reviewer: ```suggestion

.EOS Notification Banner
```

When either the end of maintenance support or the end of extended lifecycle support approaches in a year, you will see a notification banner in the {ProjectWebUI} if you have hosts with that {EL} version.
The notification provides information about the {EL} version, the number of hosts running that version in your environment, the lifecycle support, and the expiration date.

> Along with other information, the {EL} lifecycle column is visible in the report.

Reviewer: ```suggestion
Along with other information, the {EL} lifecycle column is visible in the report.
```
What report?

Author: That should be "notification". Changing from "report" to "notification".

> For any hosts that are not running {EL}, *Unknown* displays in the *RHEL Lifecycle status* and *Last report* columns.

Reviewer: ```suggestion
For any hosts that are not running {EL}, *Unknown* displays in the *RHEL Lifecycle status* and *Last report* columns.
```
I think this sentence should go to the module, where the Hosts index page is mentioned.
